### ISSUE TITLE
syntax highlighter works incorrect sometimes

### ISSUE THREAD

**Issue by leftstick** [pedronauck/docz#687]
## Bug Report

**Describe the bug**

Following code block displayed incorrectly sometimes

````bash
### yarn

```bash
yarn add axios-last
```

### npm

```bash
npm i axios-last
```
````

Usually it works fine as below:

![屏幕快照 2019-03-17 下午5 38 26](https://user-images.githubusercontent.com/1059956/54488340-87821e00-48db-11e9-95e9-14dead135e91.png)

**To Reproduce**

1. Go to [https://leftstick.github.io/axios-last/#/#installation](https://leftstick.github.io/axios-last/#/#installation)

The code blocks seems fine, but sometimes it shows as below:

![屏幕快照 2019-03-17 下午5 30 54](https://user-images.githubusercontent.com/1059956/54488235-6ff66580-48da-11e9-82db-79a29f6e15f3.png)

You can try to open the page via phone to see this issue

*In order to allow us to quickly reproduce you issue please include a link to a reproduction repository.*

repo: [https://github.com/leftstick/axios-last](https://github.com/leftstick/axios-last)

**Expected behavior**

All code blocks display without any issue.

**Environment**

* OS: macOS Mojave 10.14.3
* Node/npm version: v11.11.0

**Comment by davixyz** [comment]
I've seen this issue as well, almost always just after I deploy a new production version of the site. I think it might be due to some caches because after a hard refresh everything appears correctly.

**Comment by leftstick** [comment]
@davixyz no luck for me, still having the issue randomly even i deleted all the cache data.

**Comment by leftstick** [comment]
fixed with `1.1.0`
